src/DropdownFilter.xml

```diff
@@ -0,0 +1,73 @@
+        </property>
+        <property key="values" type="object" isList="true" required="false">
+            <caption>Options</caption>
+            <category>Data source</category>
+            <description>The filter options</description>
+            <properties>
+                <property key="caption" type="translatableString" required="false">
+                    <caption>Caption</caption>
+                    <category>General</category>
+                    <description/>
+                    <translations>
+                        <translation lang="en_US">Caption</translation>
+                        <translation lang="nl_NL">Onderschrift</translation>
+                    </translations>
+                </property>
+                <property key="comparison" type="enumeration" defaultValue="static">
+                    <caption>Comparison</caption>
+                    <category>General</category>
+                    <description/>
+                    <enumerationValues>
+                        <enumerationValue key="static">Static</enumerationValue>
+                        <enumerationValue key="XPath">XPath</enumerationValue>
+                    </enumerationValues>
+                </property>
+                <property key="attribute" type="attribute" required="false" entityProperty="../entity">
+                    <caption>Attribute</caption>
+                    <category>General</category>
+                    <description>Attribute on the list view datasource entity or list view filter entity</description>
+                    <attributeTypes>
+                        <attributeType name="String"/>
+                        <attributeType name="Integer"/>
+                        <attributeType name="Decimal"/>
+                        <attributeType name="Boolean"/>
+                    </attributeTypes>
+                </property>
+                <property key="value" type="string" required="false">
+                    <caption>Value</caption>
+                    <category>General</category>
+                    <description/>
+                </property>
+                <property key="constraint" type="entityConstraint" required="false" entityProperty="../entity">
+                    <caption>Constraint</caption>
+                    <category>General</category>
+                    <description/>
+                </property>
+                <property key="filterMethod" type="enumeration" defaultValue="contains">
+                    <caption>Filter method</caption>
+                    <category>General</category>
+                    <description>Contains as default and first option</description>
+                    <enumerationValues>
+                        <enumerationValue key="equals">Equals</enumerationValue>
+                        <enumerationValue key="contains">Contains</enumerationValue>
```

Comment:
The defaultValue is contains, than the first item in the enumeration should be contains

Reply:
Done